packages/astro/src/assets/fonts/utils.ts

```diff
@@ -316,3 +302,12 @@ export function familiesToUnifontProviders({
 
 	return { families, providers };
 }
+
+/**
+ * Checks if the name starts with --, doesn't include a space nor a colon.
+ * We are not trying to recreate the full CSS spec about indents:
+ * https://developer.mozilla.org/en-US/docs/Web/CSS/ident
+ */
+export function isValidCssVariableName(name: string) {
```

Comment:
I don't see any particular reason to have this function here, since it's **only** used inside the refinement. It can be inlined inside the validation, so us developers don't need to jump from one file to another just to see the logic.

Reply:
That's to be able to test it easily but I could test it in `config-validate.test.js` if you prefer

Reply:
It makes sense, since we already do that